app/react-transition/connectWithStore.jsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React from "react";
 import { connect } from "react-redux";
+import angular from "angular";
 
 const connectWithStore = (WrappedComponent, ...args) => {
   const ConnectedWrappedComponent = connect(...args)(WrappedComponent);
 
   const ConnectedComponentWithStore = props => {
     // Get the injector + store at render time
     // Otherwise it happens before the angular app is set up
-    const injector = angular.element(document).injector();
-    const store = injector.get("$ngRedux");
-    return <ConnectedWrappedComponent store={store} {...props} />;
+    if (!props.store) {
+      const injector = angular.element(document).injector();
+      /* eslint-disable no-param-reassign */
```

Comment:
I couldn't figure out how to do this so that the linter wouldn't get mad. @rjmackay do you know what to do here?